packages/ses/src/module-instance.js

```diff
@@ -73,6 +73,41 @@ export const makeThirdPartyModuleInstance = (
         update(value);
       };
     });
+    // Default was not in exports, need an implicit default
```

Comment:
imports function wants it, but I'm not sure if adding the default will not produce artifacts elsewhere. Tests seem to be fine.